include/quicly.h

```diff
@@ -345,7 +346,7 @@ struct st_quicly_conn_streamgroup_state_t {
          */                                                                                                                        \
         uint64_t received;                                                                                                         \
         uint64_t sent;                                                                                                             \
-    } num_bytes
+    } num_bytes;
```

Comment:
I do not think we need a semicolon here, as the macro is expected to be followed by a semicolon.

Reply:
Yup, agreed, I just had it for symmetry :-) Removed.